kotlin/find-address/src/main/java/com/esri/arcgisruntime/sample/findaddress/MainActivity.kt

```diff
@@ -212,12 +210,12 @@ class MainActivity : AppCompatActivity() {
    */
   private fun displaySearchResultOnMap(geocodeResult: GeocodeResult) {
     // clear graphics overlay of existing graphics
-    graphicsOverlay?.graphics?.clear()
+    graphicsOverlay.graphics?.clear()
```

Comment:
I think you can get rid of the `?` here as graphics is no longer at risk of being null